terraform/cloud-platform-components/external-dns.tf

```diff
@@ -66,12 +66,6 @@ podAnnotations:
 EOF
   ]
 
-  depends_on = [
```

Comment:
IIRC, these were added in order to get the components to install in a single pass. 

What's the reason for removing the "depends_on" here?

Reply:
Good catch Took them out for testing. Added back in now